### PR TITLE
[hotfix][ray] fix ray dataset compatibility

### PR DESCRIPTION
**Pull request by chaokunyang**
## What do these changes do?
Ray dataset in ray 1.10.0 has an incompatible change and it's not easy to fix unless we rewrite the dataset integration using datasource api of ray dataset. So we downgrade ray version to 1.9 and will relax version when we finished the rewrite.

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://docs.pymars.org/en/latest/development/contributing.html#check-code-styles) for how to run them

### CODE REVIEW

**Review by qinxuye — DISMISSED** [review]
LGTM

**Comment by qinxuye** [comment]
Typo needs to be fixed, see https://github.com/mars-project/mars/pull/2693/checks?check_run_id=5135822872

File: .github/workflows/platform-ci.yml [hunk 1]
```diff
@@ -82,7 +82,8 @@ jobs:
               rm -fr /tmp/etcd-$ETCD_VER-linux-amd64.tar.gz /tmp/etcd-download-test
             fi
             if [ -n "$WITH_RAY" ]; then
-              pip install pip install ray[default]
+              # relax version after refactoring ray dfataset integration useing ray dataset source api.
```
Comment (zhongchun) [inline .github/workflows/platform-ci.yml:85]: typo: `useing` should be `using`

Comment (yuyiming) [reply .github/workflows/platform-ci.yml:85]: It is better to add a `TODO:`

Comment (chaokunyang) [reply .github/workflows/platform-ci.yml]: done

File: .github/workflows/platform-ci.yml [hunk 2]
```diff
@@ -82,7 +82,8 @@ jobs:
               rm -fr /tmp/etcd-$ETCD_VER-linux-amd64.tar.gz /tmp/etcd-download-test
             fi
             if [ -n "$WITH_RAY" ]; then
-              pip install pip install ray[default]
+              #TODO relax version after refactoring ray dataset integration using ray datasource api.
```
Comment (yuyiming) [inline .github/workflows/platform-ci.yml:85]: Add a space between `#` and `TODO`, and do not forget a `:`

**Review by yuyiming — APPROVED** [review]

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**